recipes/rocksdbdict/meta.yaml

```
@@ -0,0 +1,47 @@
+{% set name = "rocksdbdict" %}
+{% set version = "0.7.1" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 457c86fc87d7060567453065e81dccbf349bb29778c933a37ed18685dd61c6cb
+
+build:
+  number: 0
+  skip: True  # [win]
```

This looks like it can be noarch?
```suggestion
  noarch: python
```
There is no need to worry about the Windows CI failing with missing dependencies as it'll only be built on Linux.